PWGJE/Tasks/jetSpectraEseTask.cxx

```diff
@@ -154,29 +186,70 @@ struct JetSpectraEseTask {
     if (!isAcceptedLeadTrack(tracks))
       return;
 
+    if (cfgEvSelOccupancy) {
+      auto occupancy = collision.trackOccupancyInTimeRange();
+      if (occupancy < cfgCutOccupancy->at(0) || occupancy > cfgCutOccupancy->at(1))
+        registry.fill(HIST("hEventCounter"), counter++);
+      return;
+    }
+
     registry.fill(HIST("hEventCounter"), counter++);
     registry.fill(HIST("hRho"), collision.rho());
     for (auto const& jet : jets) {
-      float jetpT_bkgsub = jet.pt() - (collision.rho() * jet.area());
+      float jetpTbkgsub = jet.pt() - (collision.rho() * jet.area());
       registry.fill(HIST("hJetPt"), jet.pt());
-      registry.fill(HIST("hJetPt_bkgsub"), jetpT_bkgsub);
+      registry.fill(HIST("hJetPt_bkgsub"), jetpTbkgsub);
       registry.fill(HIST("hJetEta"), jet.eta());
       registry.fill(HIST("hJetPhi"), jet.phi());
       registry.fill(HIST("hJetArea"), jet.area());
 
       float dPhi = RecoDecay::constrainAngle(jet.phi() - vPsi2, -o2::constants::math::PI);
       registry.fill(HIST("hdPhi"), dPhi);
-      registry.fill(HIST("hJetPtdPhiq2"), jetpT_bkgsub, dPhi, qPerc[0]); /* check the dphi */
+      registry.fill(HIST("hCentJetPtdPhiq2"), collision.centrality(), jetpTbkgsub, dPhi, qPerc[0]);
     }
     registry.fill(HIST("hEventCounter"), counter++);
+
+    if (collision.centrality() < centRange->at(0) || collision.centrality() > centRange->at(1)) /* for counting */
+      return;
+    registry.fill(HIST("hEventCounter"), counter++);
   }
   PROCESS_SWITCH(JetSpectraEseTask, processESEDataCharged, "process ese collisions", true);
 
+  void processESEOccupancy(soa::Join<aod::JetCollisions, aod::Qvectors, aod::QPercentileFT0Cs>::iterator const& collision,
+                           soa::Join<aod::JetTracks, aod::JTrackPIs> const& tracks,
+                           soa::Join<aod::Tracks, aod::TracksExtra, aod::TracksDCA, aod::TrackSelection> const&)
```

Comment:
why do you need the original track table in this process function?

Reply:
Hi Nima, I added it in case I needed something from there. I suppose I forgot to remove it again. I will make another commit!